src/umpire/device_allocator_helper.cpp

```diff
@@ -96,17 +106,25 @@ __host__ __device__ bool is_device_allocator(int id)
 //////////////////////////////////////////////////////////////////////////
 __host__ DeviceAllocator make_device_allocator(Allocator allocator, size_t size, const std::string& name)
 {
-  static size_t allocator_id{0};
-  auto dev_alloc = DeviceAllocator(allocator, size, name, allocator_id);
+  static int allocator_id{0};
+  int da_id = ((-1) * (allocator_id)) - 1;
+
+  // The DA ID should not conflict with other allocator IDs,
```

Comment:
Please change "DA" to "Device Allocator" or "DeviceAllocator" in documentation and comments.